code/camera/camera.cpp

```diff
@@ -71,11 +74,48 @@ auto FovOption = options::OptionBuilder<float>("Graphics.FOV",
 					      return true;
 					 })
 					 .display(fov_display)
-					 .default_val(0.75f)
+					 .default_val(fov_default)
 					 .level(options::ExpertLevel::Advanced)
```

Comment:
Why not use `DEFAULT_FOV` here?

Reply:
Options Builder wouldn't accept the define as a parameter causing builds to fail. I'm not sure why exactly.